modules/dlm/qa/with-security/src/javaRestTest/java/org/elasticsearch/dlm/PermissionsIT.java

```diff
@@ -0,0 +1,146 @@
+    }
+
+    @Override
+    protected Settings restAdminSettings() {
+        String token = basicAuthHeaderValue("test_admin", new SecureString("x-pack-test-password".toCharArray()));
+        return Settings.builder().put(ThreadContext.PREFIX + ".Authorization", token).build();
+    }
+
+    private Settings restUnprivilegedClientSettings() {
+        // Note: This user is defined in build.gradle, and assigned the role "not_privileged". That role is defined in roles.yml.
+        String token = basicAuthHeaderValue("test_non_privileged", new SecureString("x-pack-test-password".toCharArray()));
+        return Settings.builder().put(ThreadContext.PREFIX + ".Authorization", token).build();
+    }
+
+    /**
+     * Tests that a policy that simply deletes an index after 0s succeeds when an index
+     * with user `test_admin` is created referencing a policy created by `test_dlm` when both
+     * users have read/write permissions on the index. The goal is to verify that one
+     * does not need to be the same user who created both the policy and the index to have the
+     * index be properly managed by DLM.
+     */
+    @SuppressWarnings("unchecked")
+    public void testManageDLM() throws Exception {
+        String dataStreamName = "dlm-test"; // Needs to match the pattern of the names in roles.yml
+        createDataStreamAsAdmin(dataStreamName);
+        Response createDatastreamRepsonse = adminClient().performRequest(new Request("GET", "/_data_stream/" + dataStreamName));
+        final List<Map<String, Object>> nodes = ObjectPath.createFromResponse(createDatastreamRepsonse).evaluate("data_streams");
+        String index = (String) ((List<Map<String, Object>>) nodes.get(0).get("indices")).get(0).get("index_name");
+
+        Request explainLifecycleRequest = new Request("POST", "/" + index + "/_lifecycle/explain");
+        Request getLifecycleRequest = new Request("GET", "_data_stream/" + dataStreamName + "/_lifecycle");
+        Request deleteLifecycleRequest = new Request("DELETE", "_data_stream/" + dataStreamName + "/_lifecycle");
+        Request putLifecycleRequest = new Request("PUT", "_data_stream/" + dataStreamName + "/_lifecycle");
+        putLifecycleRequest.setJsonEntity("{}");
+
+        makeRequest(client(), explainLifecycleRequest, true);
+        makeRequest(client(), getLifecycleRequest, true);
+        makeRequest(client(), deleteLifecycleRequest, true);
+        makeRequest(client(), putLifecycleRequest, true);
+
+        try (RestClient nonDlmManagerClient = buildClient(restUnprivilegedClientSettings(), getClusterHosts().toArray(new HttpHost[0]))) {
+            makeRequest(nonDlmManagerClient, explainLifecycleRequest, true);
+            makeRequest(nonDlmManagerClient, getLifecycleRequest, true);
+            makeRequest(nonDlmManagerClient, deleteLifecycleRequest, false);
+            makeRequest(nonDlmManagerClient, putLifecycleRequest, false);
+        }
```

Comment:
Would be great to cover a 403 scenario for a data stream not covered by index pattern in role, e.g.,:

```suggestion
        String otherDataStreamName = "other-dlm-test";
createDataStreamAsAdmin(otherDataStreamName);
Response getOtherDataStreamResponse = adminClient().performRequest(new Request("GET", "/_data_stream/" + otherDataStreamName));
final List<Map<String, Object>> otherNodes = ObjectPath.createFromResponse(getOtherDataStreamResponse).evaluate("data_streams");
String otherIndex = (String) ((List<Map<String, Object>>) otherNodes.get(0).get("indices")).get(0).get("index_name");
Request putOtherLifecycleRequest = new Request("PUT", "_data_stream/" + otherDataStreamName + "/_lifecycle");
putOtherLifecycleRequest.setJsonEntity("{}");
makeRequest(client(), new Request("POST", "/" + otherIndex + "/_lifecycle/explain"), false);
makeRequest(client(), new Request("GET", "_data_stream/" + otherDataStreamName + "/_lifecycle"), false);
makeRequest(client(), new Request("DELETE", "_data_stream/" + otherDataStreamName + "/_lifecycle"), false);
makeRequest(client(), putOtherLifecycleRequest, false);
        }
```